dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -90,8 +90,16 @@ void BlobStore::registerPaths()
             if (blob_id != INVALID_BLOBFILE_ID)
             {
                 Poco::File blob(fmt::format("{}/{}", path, blob_name));
-                delegator->addPageFileUsedSize({blob_id, 0}, blob.getSize(), path, true);
-                blob_stats.createStatNotChecking(blob_id, lock_stats);
+                auto blob_size = blob.getSize();
+                delegator->addPageFileUsedSize({blob_id, 0}, blob_size, path, true);
+                if (blob_size > config.file_limit_size)
+                {
+                    blob_stats.createBigStat(blob_id, lock_stats);
```

Comment:
seems it will move `roll_id` forward and make BlobStore not the same as the last?

Reply:
make it behavior like `createBigStatNotChecking`

Reply:
split it into `createBigStat` and `createBigPageStatNotChecking`.